Refactor tanstack

eldu opened the pull request:
closes #51

eldu commented:
- [ ] Center headers and cells vertically
- [ ] Sort icon generic
- [ ] Hyperlink title
- [ ] Remove URL

hetd54 commented:
A couple of data things you might not have control over, but I will flag:

Some, but not all author lists end in a comma:
<img width="1625" alt="Screenshot 2024-03-05 at 11 20 51 AM" src="https://github.com/brown-ccv/ccv-pubs-frontend/assets/35639529/686b1864-9f8c-4ff9-b450-81be601a0df1">

Some title's have html tags within them:
<img width="1642" alt="Screenshot 2024-03-05 at 11 20 21 AM" src="https://github.com/brown-ccv/ccv-pubs-frontend/assets/35639529/172572a5-767b-40ec-85d8-559b7ac4aa5c">

Some pubs appear more than once:
<img width="1662" alt="Screenshot 2024-03-05 at 11 19 45 AM" src="https://github.com/brown-ccv/ccv-pubs-frontend/assets/35639529/aad879bc-f14b-409c-8757-d637c3ae4c12">

hetd54 commented on src/styles/custom.scss:
```diff
@@ -1,18 +1,44 @@
 
 .form-group.required > .form-label:after {
-  content: '*';
+  content: ' *';
```
For my own understanding, what does adding a space do here?

eldu replied:
So if you open up the form modal, all the required fields have " *" next to it if it's a required field. The space in this case is just a space. So instead of `DOI*` we get `DOI *`

<img width="566" alt="Screenshot 2024-03-05 at 4 42 23 PM" src="https://github.com/brown-ccv/ccv-pubs-frontend/assets/5270855/c7d38695-18aa-4ea7-bc9c-781aeb2eb64c">

eldu commented:
@hetd54 

> A couple of data things you might not have control over, but I will flag:
> 
> Some, but not all author lists end in a comma: 

I added a comment on #71 
There is a bunch of weird things with the data. Tim also pointed out #56 


> Mobile issues
> 
> I cannot tap into the year filters on mobile.
> 
> Also on mobile, the chart's y axis labels get chopped off (also tested on my actual phone to make sure): 
> 
> Otherwise looks great to me!

I added a new issue #75 ! Thanks for pointing that out :)
